web-app/src/pages/apply-to-job-ad/company-id-from-job-ad-id.graphql

```diff
@@ -0,0 +1,7 @@
+query companyIdFromJobAdId($jobAdId: ID!) {
+	jobAd(id: $jobAdId){
```

Comment:
Is this the best way to get the id of a company from a job ad id? Is it necessary / best practice to create a separate file for a query if the query is really small like this one?

Reply:
Yes, this is the way to get the id (or any other data) of a thing that is related to another thing. However, it would be better to change the `ApplyToJobAdPayload` so that it returns the job ad that was applied to so that we don't need a separate query for this information. Lets do that on a separate pull request.

I would recommend to always put GraphQL queries, mutations, fragments, etc. in `.graphql` files, even if they are small. It's easier to set up the tools we use if we just do them all the same way. That said, each `.graphql` file can have more than one thing in it. You should put this query in the same file as the `mutation applyToJobAd` because they are used by the same component.